### PR TITLE
Added formatting to download data to force x's and c's in place of NA…

### PR DESCRIPTION
**Pull request by rmbielby**
Updated the formatting of the downloaded data to properly use c's and x's in place of -10000 and NA.

### CODE REVIEW

File: R/subject_industry.R [hunk 1]
```diff
@@ -240,11 +240,6 @@ crosstab_text <- function(tables_data_grouped, subjectinput, YAGinput, countinpu
     subjecttext <- "all subjects",
     subjecttext <- subjectinput
   )
-  print("Here's the output I to check on the null data test:")
```
Comment (cjrace) [inline R/subject_industry.R:243]: Do we still need these console outputs?

File: R/subject_industry.R [hunk 2]
```diff
@@ -1164,8 +1159,6 @@ crosstab_text <- function(tables_data_grouped, subjectinput, YAGinput, countinpu
       )
     }
   }
-  print(crosstab_text)
```
Comment (cjrace) [inline R/subject_industry.R:1167]: Console output?

File: server.R [hunk 3]
```diff
@@ -190,9 +190,6 @@ server <- function(input, output, session) {
 
   output$crosstab_text <- renderText({
     x <- reactiveSubjbyIndText()
-    print("Here's the reactive summary text...")
```
Comment (cjrace) [inline server.R:193]: Console output?

File: server.R [hunk 4]
```diff
@@ -271,6 +268,11 @@ server <- function(input, output, session) {
       if (input$earningsbutton == "Proportions") {
         dfDownload <- dfDownload %>%
           mutate_if(is.numeric, funs(100.0 * .))
+      } else {
+        dfDownload <- dfDownload %>%
+          mutate_if(is.numeric, funs(gsub(" ", "", format(., scientific = FALSE)))) %>%
+          mutate_all(funs(gsub("-10000", "c", .))) %>%
+          mutate_all(funs(ifelse(. == "NA", "x", .)))
```
Comment (cjrace) [inline server.R]: Is there an alternative to funs() we should be using (I think I've seen that it's deprecated?)

**Review by cjrace — CHANGES_REQUESTED** [review]
Still getting NAs in the download - On subject by industry, select ethnicity > physics and astronomy.

image

File: server.R (hunk 4 above)
Comment (rmbielby) [reply server.R]: Yep, did look at that separately yesterday elsewhere in the code. Think there were a few options. Will pick one and apply in this case.

**Comment by cjrace** [comment]
Still seems broken, though in a slightly different way, x's are mostly appearing where there were NAs though some of those are now showing as NaN instead? (Same scenario - on subject by industry, select ethnicity > physics and astronomy)

image

File: server.R (hunk 4 above)
Comment (rmbielby) [reply server.R]: Have switched these instances from funs() to list(~) now.

**Comment by rmbielby** [comment]
> 

The NaNs look like they originate in the issue that Charlotte raised yesterday whereby proportions should be 0 when we're calculating a proportion of 0/0. They're not doing that in the nested table data at least, so reach the download point as NaNs. In the online tables, they're being processed as x's instead. 

So it's a double bug in that the download gives a Nan and the online tables give and x, but both should be 0.

**Review by cjrace — APPROVED** [review]

**Pull request merged**